README.md

```diff
@@ -27,7 +27,39 @@ Notebooks uses AI Platform version 1.14, we had to code in Python 3.5 (no
 f-strings or type hinting)
 
 ### Jupyter via remote execution into AI Platform Notebook Instance
-Coming Soon - Some documentation already complete by team
+Prerequisites: 
+1. You must have Anaconda/Jupyter Notebook installed 
```

Comment:
I think it would be clearer to say that you must have python (I think 3.5+) with jupyter